src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const SUM = days * PRICE_PER_DAY;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
```

Comment:
The use of constants for 'LONG_TERM', 'LONG_TERM_DISCOUNT', 'SHORT_TERM', and 'SHORT_TERM_DISCOUNT' is good as it avoids hardcoding and makes the values easily changeable and understandable.